src/chapters/booleans-and-conditionals/conditionals.rst

```diff
@@ -0,0 +1,233 @@
+
+We summarized the condition as follows: Send a notification of an upcoming bill if the statement "the bill is unpaid" is true.
+
+In such a program, booleans would allow us to write JavaScript that represents the conditional "the bill is unpaid", and based on the truth of such a statement, execute or skip code that notifies the user of an upcoming due date.
+
+The JavaScript construct that enables such behavior is a **conditional**.
+
+``if`` Statements
+-----------------
+
+.. index:: ! if, code block
+
+The most basic form of a conditional is an **if statement**. Here's how to create one in JavaScript:
+
+.. figure:: figures/if.png
+   :height: 400px
+   :alt: The structure of a conditional with an if statement
+
+Let's look at each component of this new syntax.
+
+- The ``if`` statement consists of a header line and a body. The header line begins with the keyword ``if`` followed by a boolean expression enclosed in parentheses. 
+- ``condition`` is a boolean expression, that is, something that evaluates to either ``true`` or ``false``.
+- The statements that follow the condition, within ``{}``, make up a **code block**. The code within the brackets will be executed when the condition evaluates to ``true``, and will be skipped otherwise.
+
+And here's an explicit example that mimics our banking example.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      let billHasBeenPaid = false;
+
+      if (!billHasBeenPaid) {
+         console.log("Your bill is due soon!");
+      }
+
+   **Output**
+
+   ::
+
+      Your bill is due soon!
+
+The message prints because ``billHasBeenPaid`` is ``false``, so ``!billHasBeenPaid`` evalutes to ``true``. If we were to change the value of ``billHasBeenPaid`` to be ``true``, then ``!billHasBeenPaid`` would evaluate to ``false`` and the code block would *not* execute.
+
+The condition in an ``if`` statement can be any boolean expression, such as ``name === 'Jack'`` or ``points > 10`` (here, ``name`` and ``points`` are variables). Additionally, the code block associated with a conditional can be of any size. This conditional has a code block with two lines of code:
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      if (num % 2 === 0 && num > 3) {
+         console.log(num, "is even");
+         console.log(num, "is greater than 3");
+      }
+
+While not required, the code within a conditional code block is typically indented to make it more readable. Similarly, it is a common convention to place the opening ``{`` at the end of the first line, and the closing ``}`` on a line of its own following the last line of the code block. 
+
+You should follow such conventions, even though not doing so won't create an error. To see why, compare the readability of this example, which is functionally equivalent to the one above.
```

Comment:
Double negative dance.  How about, "You should follow such conventions, even though ignoring them will not create an error."